flink-runtime/src/main/java/org/apache/flink/runtime/highavailability/JobResultStoreOptions.java

```diff
@@ -65,4 +65,11 @@ public class JobResultStoreOptions {
                                     + "are, instead, marked as clean to indicate their state. In this "
                                     + "case, Flink no longer has ownership and the resources need to "
                                     + "be cleaned up by the user.");
+
+    @Documentation.Section(Documentation.Sections.COMMON_HIGH_AVAILABILITY_JOB_RESULT_STORE)
+    public static final ConfigOption<Integer> TIME_TO_REMOVE_CLEAN_JOB_RESULT =
+            ConfigOptions.key("job-result-store.ttl-clean-job-result")
```

Comment:
The parameter name doesn't specify the embedded job result store as target component. Users who use the file-based component might be surprised that this option doesn't have any effect.

But this keeps me wondering: Shouldn't we also offer this functionality to the file-based `JobResultStore`? WDYT? This allows for automatic removal of files after a certain amount of time. Sticking to the default value being set to `infinity` wouldn't change the current behavior.

Reply:
> But this keeps me wondering: Shouldn't we also offer this functionality to the file-based JobResultStore?

i can completely resonate with it. we should definately extend it for file-based `JobResultStore`. I will make changes for it in this PR itself.